common/dynamic_country_names/imperia_dynamic_country_names.txt

```diff
@@ -1,217 +1,210 @@
-
-		is_main_tag_only = yes # default no, if yes then only the primary country for a definition can use this name (ie Poland, not dynamic Silesia-Poland)
-		priority = 1 # default 0, if multiple names have valid triggers, higher priority is used. If same priority, first one in file is used
-		
-		trigger = { # Scripted trigger, country-scope
+		name = xib_shaan_gan		# This can be left empty, and then default tag name will be used
+		adjective = xib_shaan_gan_adj		# This can be left empty, and then default tag adjective will be used
+		is_main_tag_only = yes		# default no, if yes then only the primary country for a definition can use this name (ie Poland, not dynamic Silesia-Poland)
+		priority = 1		# default 0, if multiple names have valid triggers, higher priority is used. If same priority, first one in file is used
+		trigger = {
+			# Scripted trigger, country-scope
 			exists = c:CHI
 			OR = {
-				c:CHI = {
-				has_diplomatic_pact = {
-					who = c:XIB
-					type = puppet
-					is_initiator = yes
+				c:CHI ?= {
+					has_diplomatic_pact = {
+						who = c:XIB
+						type = puppet
+						is_initiator = yes
 					}
 				}
-				c:CHI = {
-				has_diplomatic_pact = {
-					who = c:XIB
-					type = vassal
-					is_initiator = yes
+				c:CHI ?= {
+					has_diplomatic_pact = {
+						who = c:XIB
+						type = vassal
+						is_initiator = yes
 					}
 				}
-				c:CHI = {
-				has_diplomatic_pact = {
-					who = c:XIB
-					type = dominion
-					is_initiator = yes
+				c:CHI ?= {
+					has_diplomatic_pact = {
+						who = c:XIB
+						type = dominion
+						is_initiator = yes
 					}
 				}
-			}			
+			}
 		}
 	}
 }
 
 HNA = {
 	dynamic_country_name = {
-		name = hna_huguang # This can be left empty, and then default tag name will be used
-		adjective = hna_huguang_adj # This can be left empty, and then default tag adjective will be used
-
-		is_main_tag_only = yes # default no, if yes then only the primary country for a definition can use this name (ie Poland, not dynamic Silesia-Poland)
-		priority = 1 # default 0, if multiple names have valid triggers, higher priority is used. If same priority, first one in file is used
-		
-		trigger = { # Scripted trigger, country-scope
+		name = hna_huguang		# This can be left empty, and then default tag name will be used
+		adjective = hna_huguang_adj		# This can be left empty, and then default tag adjective will be used
+		is_main_tag_only = yes		# default no, if yes then only the primary country for a definition can use this name (ie Poland, not dynamic Silesia-Poland)
+		priority = 1		# default 0, if multiple names have valid triggers, higher priority is used. If same priority, first one in file is used
+		trigger = {
+			# Scripted trigger, country-scope
 			exists = c:CHI
 			OR = {
-				c:CHI = {
-				has_diplomatic_pact = {
-					who = c:HNA
-					type = puppet
-					is_initiator = yes
+				c:CHI ?= {
+					has_diplomatic_pact = {
+						who = c:HNA
```

Comment:
c:YUE should be scope:actor and you should check whether it exists at the beginning (like c:HNA)